charactersheet/charactersheet/viewmodels/character/status_line.js

```diff
@@ -34,7 +34,14 @@ function StatusLineViewModel() {
         if (!profile || statuses.length == 0) { return ''; }
 
         return profile.characterName() + ' is ' + statuses.map(function(e, i, _) {
-            return '<span class="text-' + e.type() + '">' + e.name() + '</span>';
-        }).join('&nbsp;,') + '.';
+            var status = '<span class="text-' + e.type() + '">' + e.name() + '</span>';
```

Comment:
My two issue with this addition is the sentence grammar. When inspired is put second, it just says 
`..., and inspired.`
Which sounds weird and incorrect. I really don't think the comma is correct per this:
> Use commas to separate three or more words, phrases, or clauses written in a series.

Like I stated in my previous comment, I think it should say
>Blah is carrying 12lbs and is inspired